cmd/stack_create_test.go

```diff
@@ -24,54 +24,54 @@ import (
 )
 
 func TestStackCreateSampleStack(t *testing.T) {
-	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
+	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, false)
```

Comment:
Does this still need to be false, or can we try it parallel now?

Reply:
There are a few that are false that could probably be changed to true now but I'd need to do some testing first. Would you want these in with this PR?

Reply:
If they work with `true` without any other changes, let's include them in this PR. Otherwise leave them `false` and we'll address in a future PR.